### PR TITLE
gitolite: fix reference to /bin

### PR DESCRIPTION
**Pull request by tomberek**
###### Description of changes

sskm command cannot clean up temp directory

###### Things done

patch a "/bin/rm" reference

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
 to the relevant packages
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)

### CODE REVIEW

File: pkgs/applications/version-management/gitolite/default.nix [hunk 1]
```diff
@@ -13,7 +13,7 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ nettools perl ];
   nativeBuildInputs = [ makeWrapper ];
-  propagatedBuildInputs = [ git ];
```
Comment (ck3d) [inline pkgs/applications/version-management/gitolite/default.nix:16]: Why did you add coreutils to propagatedBuildInputs? coreutils is per default available in build environments.
I have to admit, I do understand why git is propagated.

Comment (tomberek) [reply pkgs/applications/version-management/gitolite/default.nix:16]: It might be superfluous to have added it here. But the intention was to ensure it is added to the runtime environment. This is probably not the right thing to do here. I’ll fix.

**Review by ck3d — APPROVED** [review]
LGTM
